docs/codeql/support/conf.py

```diff
@@ -41,9 +41,12 @@
 
 # The version info for this project, if different from version and release in main conf.py file.
 # The short X.Y version.
-version = u'1.27'
-# The full version, including alpha/beta/rc tags.
-release = u'1.27'
+
+# LGTM Enterprise release
+release = u'1.29'
```

Comment:
Both `docs\codeql\codeql-cli\getting-started-with-the-codeql-cli.rst` and `docs\codeql\codeql-for-visual-studio-code\setting-up-codeql-in-visual-studio-code.rst` mention version 1.27. I know that this version is mentioned in examples, but mentioning this here just in case.

Reply:
Thank you for checking on this. I don't think we need to update those examples.